src/executeQuery.js

```diff
@@ -33,7 +34,7 @@ function limitQuery (features, query, options) {
     options.limit += options.offset
   }
   features.some((feature, i) => {
```

Comment:
if we're not using `i` anymore let's remove it as an argument

Reply:
did that in a subsequent commit.